src/Common/src/System/Net/Shims/DBNull.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// IMPORTANT: This is temporary code.
```

Comment:
What is the temporary-ness until? Is there something more we can note here about when it should be removed, or tying it to an issue number, or something like that?